packages/typespec-python/src/emitter.ts

```diff
@@ -72,18 +75,26 @@ export async function $onEmit(context: EmitContext<PythonEmitterOptions>) {
     const yamlMap = emitCodeModel(sdkContext);
     addDefaultOptions(sdkContext);
     const yamlPath = await saveCodeModelAsYaml("typespec-python-yaml-map", yamlMap);
-    const commandArgs = [
-        `${root}/scripts/run-python3.cjs`,
-        `${root}/scripts/run_tsp.py`,
-        `--output-folder=${outputDir}`,
-        `--cadl-file=${yamlPath}`,
-    ];
+    const platform = os.platform();
+    var venv_path = "unknown";
+    switch(platform){
```

Comment:
Should we have `default`? Would `bin` be a reasonable default (even if we're not sure)